### PR TITLE
Lavaland Monster Sprite Update

### PR DESCRIPTION
**Pull request by Mey-Ha-Zah**
Updates most Lavaland mob sprites, merges some icon files and adjusts the pathing.
## About The Pull Request

Updates Most Lavaland Mob Sprites with work provided by Sheets.
Merges, splits, and moves a few icon files and icons to adjust for the new icons.
Repaths the dmi locations and pixel shifts for the adjusted sprites.

Examples:
image
image
image

## Why It's Good For The Game

Better quality sprites are better to look at.
Putting sprites that are the same type in the same file is much saner.

## Changelog
:cl:
imageadd: added new lavaland sprites, moved some sprites to a new location, adjusted pixel shift for the updated sprites, adjusted the name of some sprites.
/:cl:

### CODE REVIEW

**Comment by Cheshify** [comment]
Can we see what they look like?

**Comment by Mey-Ha-Zah** [comment]
yes, I'm gonna add some icons in a minute

File: icons/mob/simple/lavaland/lavaland_monsters.dmi
Comment (Sealed101) [inline icons/mob/simple/lavaland/lavaland_monsters.dmi:1]: mapping helper's sprite was removed but not replaced
image

**Comment by Sealed101** [comment]
fuckin' baller sprites, sheets is putting in the good work.
also most crusher trophies would need to be updated as to not fall behind the new n flashy bitz

**Comment by IndieanaJones** [comment]
Holy shit, I love the elite sprites.

**Review by Imaginos16 — APPROVED** [review]
Holy based

**Comment by Helg2** [comment]
Color of dragon horns kinda blend in with the floor color.

**Comment by AverageUser67** [comment]
*inhale
MERGE!!!!
(after updating hierophants arena)

**Comment by Wallemations** [comment]
I heard some of these were animated but I can't see them on mobile, could you show those animated gif versions?

**Comment by LSommers** [comment]
Old Heiro looked like an animated, clearly artificial club
This just looks like a generic... something.

Ash Drake is now a dragon from the hit Bethesda title Skyrim by Todd Howard, I can't wait to buy it again (this is a joke and a complain about how you took it from four limbs + wings classic style of dragon to the post-skyrim "wings as front limbs" deal)

Watcher looks fine.

No idea what else was changed and can't be assed to figure out how to look at it. Not a fan overall.

**Comment by Jacquerel** [comment]
The checks tab (icondiffbot) will show any animated changes, though as Wallem said it usually doesn't function for me on mobile.
This is the animated Hierophant.
image

**Comment by Mey-Ha-Zah** [comment]
> Old Heiro looked like an animated, clearly artificial club This just looks like a generic... something.
> 
> Ash Drake is now a dragon from the hit Bethesda title Skyrim by Todd Howard, I can't wait to buy it again (this is a joke and a complain about how you took it from four limbs + wings classic style of dragon to the post-skyrim "wings as front limbs" deal)
> 
> Watcher looks fine.
> 
> No idea what else was changed and can't be assed to figure out how to look at it. Not a fan overall.

The drake change was inspired by an artwork piece, and the heirophant is part of something larger, just wait and see.

**Comment by Helg2** [comment]
> No idea what else was changed and can't be assed to figure out how to look at it.

https://github.com/tgstation/tgstation/pull/74299/checks
Click on the details.

**Comment by Thunder12345** [comment]
Drake should spread its wings when flying, it's not a hovercraft

**Comment by Sealed101** [comment]
> (why is his eye now a mouth?) 

retconned lmao

**Comment by Mey-Ha-Zah** [comment]
> Drake should spread its wings when flying, it's not a hovercraft

Might ask for a sprite for this in a future PR. But that is for later.

**Review by Fikou — CHANGES_REQUESTED** [review]
sentient_creature.dm renders the pandora box now, because the broodmother icon is missing, you need to either make this intentional by replacing the broodmother with pandora there, or use the icon file that has the broodmother.

I also worry about possible balance implications of how small baby goliaths are, and that brimdemons are much less expressive in their attack. (also the beam ending looks weird but thats just personal opinion)

lastly idk if you forgot or if they arent made but herald and ancient goliaths are missing directionals unlike everything else (goliaths used to have em)

**Comment by Mey-Ha-Zah** [comment]
I'll discuss the sprite changes with Sheets, and as for the sentient creature dm, I'll deal with that... if I can figure out how.

**Comment by Jacquerel** [comment]
> I'll discuss the sprite changes with Sheets, and as for the sentient creature dm, I'll deal with that... if I can figure out how.

That change is simply in `sentient_creature.dm`, another instance of "the file or icon state name changed".

**Comment by Shadowflame909** [comment]
Hey that's pretty good!

**Comment by Mey-Ha-Zah** [comment]
Alright, might be a week before I get the sprites you want Fikou.
I'm open to ideas about changes, some good ones I've seen on discord are easy enough I could do them myself.
If you have anything that's a minor nitpick (And not changing a large part of the sprite) or a small idea that can easily be done, just type it out, or message me on discord.

**Comment by Pumpkinoe** [comment]
these are awesome, great sprites
looks like the hierophant club sprite remains unchanged though?

**Comment by Furrior** [comment]
Will Hiero's staff be resprited too to match new style?

**Comment by Mey-Ha-Zah** [comment]
> Will Hiero's staff be resprited too to match new style?

All items will be updated in a future PR. This is specifically the Creatures themselves.

**Comment by Helg2** [comment]
> All items will be updated in a future PR. This is specifically the Creatures themselves.

That's just half done work then.

**Review by Fikou — APPROVED** [review]
code looks good

**Review by Imaginos16 — APPROVED** [review]
Yeah checked again since apparently one approval by yours truly wasn't enough. Still good just merge it already.

**Comment by Toastgoats** [comment]
All of these are really cool but the ash drake looks like a sprite rip from a completely different game, I'd really prefer it if you did a revamp like the other megafauna sprites instead of this.

**Pull request merged**